react/controls/TrackControl.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { RefObject, useRef, FunctionComponent, ChangeEvent } from 'react'
+
+import styles from '../styles/styles.css'
+import useVideoTrack, { formatTime } from '../useVideoTrack'
+
+export interface TrackControlProps {
+  videoRef: RefObject<HTMLVideoElement>
+  changeState: (property: string, value: any) => void
+  cssHandles: Record<'trackContainer' | 'trackTimer' | 'trackBar', string>
+}
+
+const TrackControl: FunctionComponent<TrackControlProps> = ({
+  videoRef,
+  changeState,
+  cssHandles,
+}) => {
+  const trackBarRef = useRef<HTMLInputElement>(null)
+  const { duration, currentTime, setCurrentTime } = useVideoTrack(
+    videoRef,
+    trackBarRef
+  )
+
+  const handleChange = (event: ChangeEvent<HTMLInputElement>) => {
+    if (videoRef?.current?.ended) {
+      changeState('isPlaying', false)
+    }
+
+    setCurrentTime(parseFloat(event?.target?.value))
+  }
+
+  return (
+    <div
+      className={`${cssHandles.trackContainer} w-100 absolute bottom-1 lh-title`}
+    >
+      {currentTime && duration ? (
+        <span className={`${cssHandles.trackTimer} ml7 white v-mid`}>
+          {`${formatTime(currentTime)} / ${formatTime(duration)}`}
+        </span>
+      ) : null}
```

Comment:
`currentTime` as `0` would resolve to `false`.

```suggestion
      {currentTime != null && duration != null && (
        <span className={`${cssHandles.trackTimer} ml7 white v-mid`}>
          {`${formatTime(currentTime)} / ${formatTime(duration)}`}
        </span>
      )}
```